packages/api/cms-api/src/mailer/delete-mail-logs.command.ts

```diff
@@ -0,0 +1,39 @@
+import { CreateRequestContext, MikroORM } from "@mikro-orm/core";
+import { InjectRepository } from "@mikro-orm/nestjs";
+import { EntityRepository } from "@mikro-orm/postgresql";
+import { Logger } from "@nestjs/common";
+import { add, Duration } from "date-fns";
+import { Command, CommandRunner, Option } from "nest-commander";
+
+import { MailerLog } from "./entities/mailer-log.entity";
+
+@Command({
+    name: "mailer:delete-mail-logs-older-than",
+    arguments: "<count> <duration>",
+    description: "Deletes mailer logs created before the given duration (years/months/weeks/days/hours/minutes/seconds)",
+})
+export class DeleteMailLogsCommand extends CommandRunner {
```

Comment:
@nsams keeping this command to allow deleting logs on demand. but probably we don't need it that often. The only use case I can imagine is to configure additional delete-times based on mail-purpose (e.g. keep logs for invoice-payment-reminder mails longer than you-are-mentioned mails)

Reply:
but that could probably even be something implemented in the application. this would only require to export the MailLog entity to allow using the MailLog repository... maybe this would be better.